doc/release/1.8.1-notes.rst

```diff
@@ -5,15 +5,52 @@ SciPy 1.8.1 Release Notes
 .. contents::
 
 SciPy 1.8.1 is a bug-fix release with no new features
-compared to 1.8.0.
+compared to 1.8.0. Notably, usage of Pythran has been
+restored for Windows builds/binaries.
 
 Authors
 =======
 
+* Maximilian Nöthe
```

Comment:
this author was added manually for critical contributions fixing the `.htaccess` doc server file following the last release with new doc theme